src/codecs/interface.ts

```diff
@@ -14,12 +14,12 @@ export interface BlockEncoder<Code extends number, T> {
  */
 export interface BlockDecoder<Code extends number, T> {
   code: Code
-  decode(bytes: ByteView<T>): T
+  decode(bytes: ByteView<T> | ArrayBufferView<T>): T
 }
 
 /**
  * An IPLD codec is a combination of both encoder and decoder.
  */
 export interface BlockCodec<Code extends number, T> extends BlockEncoder<Code, T>, BlockDecoder<Code, T> {}
 
-export type { ByteView }
+export type { ArrayBufferView, ByteView }
```

Comment:
We are exporting this type twice from two separate files. 

1. `src/blocks/interface -> src/codecs/interface -> src/interface`
1. `src/blocks/interface -> src/interface`

Do we need a central location or will TS handle that well?

Reply:
TS handles it fine, but it makes linking to a canonical documentation location in generated TypeDocs a little more fiddly.

Ref: https://github.com/TypeStrong/typedoc/issues/2125 and https://github.com/ipfs/aegir/blob/master/src/docs/type-indexer-plugin.js#L121-L128.